layout/_partials/post/post-meta.njk

```diff
@@ -83,11 +83,11 @@
 
   {{- next_inject('postMeta') }}
 
+  {%- if theme.symbols_count_time.separated_meta and config.symbols_count_time.symbols and config.symbols_count_time.time %}
```

Comment:
Why? I think the last two condition should be removed, like the original logic.

Reply:
If removed, an extra `post-meta-break` element will be inserted even if `symbols_count_time` is disabled

Reply:
So this is actually a new style, instead of a simple refactoring? I haven't had time (for one or two weeks) to test this so I went though these code only by the eyes. Just ignore me if you find me saying something completely irrelevant.

Reply:
The "new style" means (if I have understood it correctly):

* when the `symbols` (the 2nd condition) is truthy and the `time` (the 3rd condition) is falsy, then treat the `symbols`' `<span>` as normal `.post-meta-item` (apply the `::before` stylesheet).
* No needless empty element will be used when both condition above are falsy.

Reply:
The new style looks the same as before. Just used some CSS tricks to replace the following content, because curlylint can't parse it
```
{% if ... %}
</div>
<div>
{% endif %}
```

Reply:
With ![truthy `symbols` and falsy `time`](https://user-images.githubusercontent.com/22674396/126887897-c8e30bd7-f027-4fba-b2c5-25ef94cb800e.png),

Before it presents like:
![Word count shown in the next line](https://user-images.githubusercontent.com/22674396/126887939-71fb5dfe-558b-40d1-8bbc-5f65c4664e14.png)

The new style:
![Word count shown in the same line](https://user-images.githubusercontent.com/22674396/126887952-79e44130-dde4-4ff7-9e23-560385168a6f.png)